gui/FunctionPlotterApp.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Graph.h"
```

Comment:
As mentioned before, I'd suggest to move includes to header files so that we do not include them multiple times.